src/assets/icons/MagnifyingGlassIcon.jsx

```diff
@@ -1,6 +1,6 @@
-const MagnifyingGlassIcon = ({className}) => {
+const MagnifyingGlassIcon = ({className, width="23", height="23"}) => {
```

Comment:
can't these be defined/imported as .svg files? not a big deal but it seems like a lot of extra work to have to make every svg a js file

Reply:
Yeah the number of icons in this repo has gotten pretty big. But I find that the react components work better with Mantine out of the box. Plus, they're good for tree shaking